### PR TITLE
Adds psuedo-classes assessments

### PR DESCRIPTION
**Pull request by argyleink**
adds 4 pop quiz questions 👍🏻

### CODE REVIEW

File: src/site/content/en/learn/css/pseudo-classes/pseudo-classes.assess.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      - content: "True"
+        rationale: "Watch for the use of a single or double `:` as a key
+        distinguishing character in the selector"
+      - content: "False"
+        rationale: "Pseudo-elements are for parts, Pseudo-classes are for
+        state."
+  - type: multiple-choice
+    cardinality: "1"
+    correctAnswers: "1"
+    stem: Like psuedo-elements, all CSS styles are available for use?
```
Comment (rachelandrew) [inline src/site/content/en/learn/css/pseudo-classes/pseudo-classes.assess.yml:21]:
```suggestion
    stem: In psuedo-elements, all CSS styles are available for use.
```

File: src/site/content/en/learn/css/pseudo-classes/pseudo-classes.assess.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+setLeader: Test your knowledge of pseudo classes
+height: unset
+tabLabel: question
+questions:
+  - type: multiple-choice
+    cardinality: "1"
+    correctAnswers: "0"
+    stem: |
+      Pseudo-classes select elements based on internal browser state, while
```
Comment (rachelandrew) [inline src/site/content/en/learn/css/pseudo-classes/pseudo-classes.assess.yml:9]: I am not a fan of this question. I had to read it several times to understand what you were asking. Can we phrase this more simply?

Pseudo-classes act as if a class has been dynamically applied to an element, while pseudo-elements act on an element itself,

**Review by rachelandrew — CHANGES_REQUESTED** [review]
Couple of suggested changes here.

File: src/site/content/en/learn/css/pseudo-classes/pseudo-classes.assess.yml (hunk 1 above)
Comment (argyleink) [reply src/site/content/en/learn/css/pseudo-classes/pseudo-classes.assess.yml:21]: This changes the focus to elements though, and the question intends to quiz on classes. The idea was to help them remember the limits of pseudo element styling and consider that pseudo classes will have the same constraints. Thoughts on how we can find a middle ground?

File: src/site/content/en/learn/css/pseudo-classes/pseudo-classes.assess.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      Pseudo-elements select browser owned parts of elements?
```
Comment (argyleink) [inline src/site/content/en/learn/css/pseudo-classes/pseudo-classes.assess.yml:10]:
```suggestion
      Pseudo-classes act as if a class has been dynamically applied to an element, while pseudo-elements act on an element itself?
```

sounds good to me @rachelandrew 👍🏻

File: src/site/content/en/learn/css/pseudo-classes/pseudo-classes.assess.yml (hunk 1 above)
Comment (rachelandrew) [reply src/site/content/en/learn/css/pseudo-classes/pseudo-classes.assess.yml:21]: Maybe don't mention pseudo-elements at all? 

You can style a pseudo-class using any available CSS properties.

Comment (argyleink) [reply src/site/content/en/learn/css/pseudo-classes/pseudo-classes.assess.yml:20]: yeah, i'm with you now. this question isnt focused. i'm gonna remove it and maybe come up with a new one!

File: src/site/content/en/learn/css/pseudo-classes/pseudo-classes.assess.yml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      - content: "`:is()`"
+        rationale: "🎉"
+      - content: "`:target`"
+        rationale: "Functional pseudo-classes have `()` after them, to indicate
+        they accept paremeters."
+      - content: "`:empty`"
+        rationale: "Functional pseudo-classes have `()` after them, to indicate
+        they accept paremeters."
+      - content: "`:not()`"
+        rationale: "🎉"
```
Comment (argyleink) [inline src/site/content/en/learn/css/pseudo-classes/pseudo-classes.assess.yml:31]: here's the new question!

**Review by una — APPROVED** [review]

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**